nvidia_nims/vista_3d_remote_nim.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ngc_api_key = os.environ.get("NGC_API_KEY")
</pre><p>It may be worth noting here that people can just paste their API keys here if they're in an environment that setting an variable isn't feasible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Ok - I am willing to add that, as long as we also note that it is bad practice :).   Too often people forget to delete the keys before checking in their code, thereby inadvertently advertising their keys.  We shouldn't encourage that practice, imo.